pkg/quota/admission/clusterresourceoverride/admission.go

```diff
@@ -48,9 +48,9 @@ func init() {
 }
 
 type internalConfig struct {
-	limitCPUToMemoryRatio     *inf.Dec
-	cpuRequestToLimitRatio    *inf.Dec
-	memoryRequestToLimitRatio *inf.Dec
+	limitCPUToMemoryRatio     float64
```

Comment:
These are single machine quantities.  We don't support > int64 resources on
a single machine.  There *should* be a quantity multiply but I'm not going
to solve that now.

On Jun 30, 2016, at 8:27 AM, Luke Meyer <notifications@github.com> wrote:

In pkg/quota/admission/clusterresourceoverride/admission.go
<https://github.com/openshift/origin/pull/9578#discussion_r69154233>:

> @@ -48,9 +48,9 @@ func init() {
>  }
>
>  type internalConfig struct {
> -	limitCPUToMemoryRatio     *inf.Dec
> -	cpuRequestToLimitRatio    *inf.Dec
> -	memoryRequestToLimitRatio *inf.Dec
> +	limitCPUToMemoryRatio     float64

I don't really have a problem with switching to float but I thought the aim
of the Resource and Quantity libs was to avoid that. Is it basically
impossible to avoid float multiplication against the latest implementation?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/9578/files/bce46eb52a81a8ad35b1e6e3c079cc4fcc8a0467..69e5ec5daf8bbc30ccc5bb3184b525fcb813c8da#r69154233>,
or mute the thread
<https://github.com/notifications/unsubscribe/ABG_p4kfBLsaVhCltxrkzrBeZdJhRfO0ks5qQ-A5gaJpZM4I-xTE>
.